### ISSUE TITLE
Typo?

### ISSUE THREAD

**Issue by zinechant** [atlas-outreach-data-tools/atlas-outreach-data-tools-framework#1]
https://github.com/atlas-outreach-data-tools/atlas-outreach-data-tools-framework/blob/7c2ed6995086b0667bf13a0da10c2997d327580b/Analysis/AnalysisHelpers.py#L34

Should it be `abs(jet.eta()) < 2.4` ?

**Comment by lynashere** [comment]
yes normally it should be `abs(jet.eta())<2.4` since abs() should be applied to the numeric value to then be compared to 2.4 not the other way round.
